README.md

```diff
@@ -4,21 +4,31 @@
 
 ## About the challenge
 
-This is a pretty simple test, in which you should be able to write a React app, using the provided screens as reference. You’ll need to make requests to an API to get some JSON content, filter the data and print it on view.
+This challenge encompasses the following skills:
 
-You can use any boilerplate/approach you prefer, but try to keep it simple. We encourage you to use your favourite tools and packages to build a solid React application.
+1. Writing a solid application
+2. Making API calls
+3. Filter data and shape it to display on the UI
+
+You can use any boilerplate/approach you prefer, but try to keep it simple. We encourage you to use your favourite tools and packages to build the best app possible.
 
 ### The app should be a movie search with two pages:
 
-* **Home page:** search field and a list with the results;
+- **Home page:** search field and a list with the results;
 
   _Think about empty state, loading state and movie not found_
 
-* **Movie page:** movie details page;
+- **Movie page:** movie details page;
 
   _Think about loading state and movie not found_
 
-The purpose of this challenge is to evaluate your HTML, CSS, Javascript and React API skills and overall approach while composing a simple web app given a set of screens and an API feed. We will also assess the HTML, CSS, and JS output.
+The purpose of this challenge is to evaluate your HTML, CSS, Javascript and API skills and overall approach while composing a simple web app given a set of screens and an API feed. We will also assess the HTML, CSS, and JS output.
+
+### What is expected
+
+- A functional app with the two screens (`Home` and `Details`);
+- A clean layout and design;
+- A way to tag movies as _favourites_ (and having them persist would be great!);
```

Comment:
Shouldn't be `having them persisted would be great`

Reply:
I don't think it really matters. Seems to be ok both ways